modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -178,7 +178,77 @@ const Example = styled('div')(
 
 ### Responsive Styles
 
-> Coming Soon (https://github.com/Workday/canvas-kit/pull/1779)
+The following Canvas Kit responsive utilities will be released as part of v8:
+
+- **ResponsiveContextProvider**: This is a React Context Provider that will take a width and an
+  optional theme prop and allow components to subscribe to context changes via the
+  `useResponsiveContext` hook.
+
+- **useResponsiveContext Hook**: This hook subscribes to the `ResponsiveContextProvider` and
+  provides helpful utilities to determine whether the current width is within a particular theme
+  breakpoint (`Small: 320px, Medium: 768px, Large: 1024px, Extra Large: 1440px`). Developers can use
+  these utilities when creating container-based responsive styles.
```

Comment:
Removing an unnecessary "Hook". I agree with Manny that providing more context on breakpoints is a good idea; even better than listing them out would be to link to the breakpoints table in the Responsive Styling doc which is more explicit about when each size is applied based on the start and end of the px ranges.

I find a single value to be insufficient to understand when a particular size is applied whereas a range is more clear (i.e., `Small (s) is applied from 320px - 767px` is better than `small: 320px`).

```suggestion
- **useResponsiveContext**: This hook subscribes to the `ResponsiveContextProvider` and provides
  helpful utilities to determine whether the current width is within a particular
  [theme breakpoint](/getting-started/for-developers/resources/responsive-styling/#canvas-kit-breakpoint-sizes).
  Developers can use these utilities when creating container-based responsive styles.
```

Note that although this link _won't_ work in Storybook due to the `#`, it _will_ work on the Canvas Site (more context [here](https://github.com/Workday/canvas-kit/pull/1827#discussion_r988387128)).

Alternatively, we could reproduce the whole passage on Canvas Kit Breakpoint Sizes from your Responsive Styling here in the Upgrade Guide...

![Screen Shot 2022-10-19 at 3 15 03 PM](https://user-images.githubusercontent.com/146020/196814874-e877c724-0189-4311-b86d-bbce867db419.png)

... but I'm not sure that's necessary.